### PR TITLE
[BOP-97] Change the getting started example to use k0s

### PR DESCRIPTION
**Pull request by nwneisen**
This change is dependent on https://github.com/MirantisContainers/boundless-cli/pull/31

This swaps the getting started section so that k0s is used by default instead of kind. The old kind example was moved into the examples directory.

### CODE REVIEW

File: website/content/docs/getting-started/create.md [hunk 1]
```diff
@@ -1,31 +1,37 @@
...
+#### Download the example blueprint
+
+Download the [example blueprint](https://github.com/mirantiscontainers/boundless/tree/main/blueprints/k0s-example/k0s-example.yaml) to your machine.
+
+This blueprint is setup to install the cluster on your local machine without the need for any additional configuration.
+
+#### Apply the blueprint
 
-* [kind](https://kind.sigs.k8s.io/docs/user/quick-start/) - required for installing a kind distribution
+Applying the blueprint will create the cluster and install the components defined in the blueprint:
```
Comment (sakshisharma84) [inline website/content/docs/getting-started/create.md]: create the cluster -> create the k0s cluster

install the components defined in the blueprint: -> install the components defined in the blueprint.

**Comment by ranyodh** [comment]
Continuing our discussion on the structure of the titles/section from: [here](https://github.com/MirantisContainers/boundless/pull/18#pullrequestreview-1798361070) and and [here](https://github.com/MirantisContainers/boundless/pull/18#issuecomment-1874187200)


**Ranyodh:**
> I was thinking we should update the structure of the docs a little for better flow of the content.
> 1. Make https://mirantiscontainers.github.io/boundless/docs/getting-started/ the home page (and possible remove/hide the current homepage: https://mirantiscontainers.github.io/boundless/)
> 2. "Getting Started" should include:
>      1. Installation
>      2. "Use with Kind cluster" (change headline)
>      3. "Use with k0s cluster in AWS (move from section "Example Blueprints" > [Create a k0s cluster in AWS using Terraform](https://mirantiscontainers.github.io/boundless/docs/examples/bootstrap-k0s/) and change headline)
>      4. "Use with existing cluster" (move from section "Example Blueprints" > [Using an existing cluster](https://mirantiscontainers.github.io/boundless/docs/examples/existing-cluster/) and change headline)

**Nick**
> My thought was to try and keep the _Getting Started_ as simple as possible so that it is for someone to try it out on their local machine. Adding all of these means someone will have to start making decisions just to try it out. I can add these other sections but maybe another option would be to add links at the end so _Getting Started_ stays simple but these other setups are still brought to their attention.

Whats confusing for me regarding the current layout is the following pages in the section [**Example Blueprint**](https://mirantiscontainers.github.io/boundless/docs/examples/):
* **Create a k0s cluster in AWS using Terraform**
* **Use boundless on an existing cluster**

These are not really example blueprints, but another platforms to install Boundless on. So, I think we should move these pages to another section.

**Comment by nwneisen** [comment]
@ranyodh

> These are not really example blueprints, but another platforms to install Boundless on. So, I think we should move these pages to another section

Would it make more sense if we called the section "Example Setups" or just "Examples". I agree they aren't really blueprints. 

I was playing with another idea of having a page that lists all of our blueprints for someone who doesn't want to read all the docs with them. Basically the equivalent of https://github.com/MirantisContainers/boundless/tree/main/blueprints built into the docs website.

**Comment by ranyodh** [comment]
> @ranyodh
> 
> > These are not really example blueprints, but another platforms to install Boundless on. So, I think we should move these pages to another section
> 
> Would it make more sense if we called the section "Example Setups" or just "Examples". I agree they aren't really blueprints.
> 
> I was playing with another idea of having a page that lists all of our blueprints for someone who doesn't want to read all the docs with them. Basically the equivalent of https://github.com/MirantisContainers/boundless/tree/main/blueprints built into the docs website.

For me, moving them into "Getting Started" makes more sense. A custom can get starting with boundless using `Kind`, `K0s` or `Existing Cluster`.

**Comment by nwneisen** [comment]
@ranyodh Done. TBH I think this draws the customer's focus away from the boundless commands and a minimal setup that they can experiment with and learn what boundless can do for them.

**Comment by vivek** [comment]
@ranyodh @nwneisen,
 
To me, home page, https://mirantiscontainers.github.io/boundless/ is great because it tells readers what boundless is about and everything else about it. I do not think we should replace it with https://mirantiscontainers.github.io/boundless/docs/getting-started/.

On `Getting Started`, current Getting started doesn't tell in simple steps how a first time user can experience boundless, to that effect I agree with the proposal outlined above:

```
"Getting Started" should include:
Installation
"Use with Kind cluster" (change headline)
"Use with k0s cluster in AWS (move from section "Example Blueprints" > [Create a k0s cluster in AWS using Terraform](https://mirantiscontainers.github.io/boundless/docs/examples/bootstrap-k0s/) and change headline)
"Use with existing cluster" (move from section "Example Blueprints" > [Using an existing cluster](https://mirantiscontainers.github.io/boundless/docs/examples/existing-cluster/) and change headline)
```

**Comment by nwneisen** [comment]
@ranyodh @vivek This PR has the changes as you've described it. If there are no other issues, we can merge it in after an approval

**Review by ranyodh — APPROVED** [review]

**Pull request merged**